recipes-kernel/kexec/kexec-tools_2.0.21.bbappend

```diff
@@ -0,0 +1,11 @@
+SUMMARY = "nilrt specific modification for kdump"
+FILESEXTRAPATHS_prepend := "${THISDIR}/kexec-tools:"
+
+SRC_URI += " file://kdump.conf \
+           "
+
+pkg_postinst_kdump() {
+#!/bin/sh
+
+sed -i 's/set otherbootargs="\(.*\)"/set otherbootargs="\1 crashkernel=128M "/' /boot/runmode/bootimage.cfg
```

Comment:
Redhat has a guideline for crashkernel reserved memory size: https://access.redhat.com/solutions/59432. But 120 MB seems to be enough for NI Linux RT targets with 4 GB of RAM. Might need to bump this for large memory targets.